Source/Scene/Cesium3DTileset.js

```diff
@@ -2466,7 +2475,11 @@ function update(tileset, frameState, passStatistics, passOptions) {
     requestTiles(tileset);
   }
 
-  updateTiles(tileset, frameState, passOptions);
+  if (frameState.passes.pick && defined(tileset.pickPrimitive)) {
+    tileset.pickPrimitive.update(frameState);
+  } else {
+    updateTiles(tileset, frameState, passOptions);
+  }
```

Comment:
Even though I originally put this code in `update` I think it actually think it belongs in `updateForPass` around line 2559, something like:

```js
  if (this.show || ignoreCommands) {
    if (frameState.passes.pick && defined(this.pickPrimitive)) {
      this.pickPrimitive.update(frameState);
    } else {
      this._pass = pass;
      tilesetPassState.ready = update(this, frameState, passStatistics, passOptions);
    }
  }
```

That way the main tileset won't try to load any tiles or update statistics.

I also confirmed that `pickPrimitive` works with other passes like the offscreen most-detailed pick pass with this [sandcastle example](http://localhost:8080/Apps/Sandcastle/index.html#c=tVVbT9swFP4rVp9SqThO0qQXCpq4SEyCDdFu00R5cBPTejh2ZDtlMPHfdxKnaSlMe6KtHNvnnO/cvpOuqUZrzh6ZRkdIskd0ygwvc/y9vvPmnbQ+nyppKZdMzzvdw7lcg5VJmWRg5KxxfQSRE8LBMn2p5JLbMqvUDgIcBaNkePhKgdqNnOBkNBwO+nsQIGkiOqXawo7KCN9rld/QDPbG2/PU20PutniWC2aYDQCxjhUXmufc8jUzmGaZt5O8e0RnM2fi/ZlLhEotxmjewdh3v2nBUuOfUUv9V/r+CbXpimWb5w9uV/V2RheC+U0Y+JdRct6Zy5euC3ETHjb2SbDXzdjFn1ZiF1GqhNJVTHq5oF4Yxz20XQiOuw5/vwLhh1egsWo3e0n3KrCCpw/XG//jtjtvChJ+VEF2aItTmjNNMbireO/AMmYsl0AjJdH4lfOWiF4QBkMShjgkUQjfKBz20EF/ECVBHy6jJCRxSJIe6pNhMBoNcBAHZBD3g2HSrYugNAe2vuPjggG95fKaA29ulBAewaQfxiROSARYSRKRQa+aqTgJB3EY9eMwjEhcXRIHzWQ20zAg90rngN3gXlGr+e8+/nx2/mX2efazLceUyiylxgpWUWGmlFhQfVJaqyS8BK6hWShXxkJV4D0gWAZdRPelTKvQvS6qK1ZxrFCGV3cGGna7VzS11LRY8fS/M3t3WMFpZkstG6oamheCXTC+XNkrCOSsicNrHXaxXTHptUFtBS46F99ClGxarJjeeXWBc9BshqDRRW0m4/dSwFa1NNh6uiV3rvjVhwnBC6N4NkYtJiQFfeX/olOMSQ9tli0SQjmF8nAqtrFUTMcnl9/OW62XZlc3tE7XZfesVD5T3macnLRWqpdOrzOpJ+x4A/SJ54XSthp3D6bcMqg8+Df+okwfYIhTYzYuJv6u6STja8Szo3f+NFAqqDEguS+FmPJnNu8cT3zQf2MqVM38r2umBX2q1FbB8aW7xBhPfDi+b2kdafeQ/wI).